README.md

```diff
@@ -3,8 +3,135 @@
+FACEBOOK_APP_ID=''
+FACEBOOK_APP_SECRET=''
+GOOGLE_APP_ID=''
+GOOGLE_APP_SECRET=''
+MANAGER_EMAIL=''
+MANAGER=''
+SUPER_ADMIN_EMAIL=''
+SUPER_ADMIN_PASSWORD=''
+SUPER_ADMIN_ROLE=''
+SUPER_ADMIN_REAL_PASSWORD='' 
+```
+    
+3. Install postgres
+
+4. When the server is running, create a database on your new PG server. Ensure your new development database is the same name as your ` barefoot_nomad_dev_db `  and your test database is the same name as ` Tbarefoot_nomad_test_db  ` environment variables
+
+5. Ensure you have Postgres running 
+
+### B. Running the application
+
+In your terminal:
+
+1. Run `npm install` to install all dependencies
+
+2. For **Testing**: run `npm run test`
+
+3. For **Development**: run `npm run dev-start`
+
+4. Access ` http://localhost:<:APPLICATION_PORT> ` (by default, the port is `3000`) in Postman, if app is running correctly, you will get a response with a message:
+
+    ```javascript
+   { 
+       "message": "Welcome to Barefoot Nomad."
+   }
+    ```
+
+
+## API Endpoints
+
+- #### Default route
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/](https://team-odd-bn-backend-staging.herokuapp.com/) | GET |
+
+- #### Sign up endpoint
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/signup](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/signup) | POST |
+
+- #### Log in endpoint
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/signin](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/signin) | POST |
+
+- #### Log in via facebook endpoint
+
+**Note** to run this endpoint you need to have access token for facebook
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/facebook](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/facebook) | POST |
+
+- #### Log in via google endpoint
+
+**Note** to run this endpoint you need to have access token for google
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/google](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/google) | POST |
+
+- #### Forgot password endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/forgot-password](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/forgot-password) | POST |
+
+- #### Reset password endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/reset-password/[token]](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/forgot-password) | PATCH |
+
+- #### Verify email endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/verify-email/[token]](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/verify-email/) | GET |
+
+- #### Resend email endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/:id/resend-email](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/:id/resend-email) | GET |
```

Comment:
This should be put in Documentation not here. Think about it,  as you progress you'll create new endpoints, will you keep adding them here?